handler/instrumented_enqueue_object.go

```diff
@@ -41,21 +41,21 @@ type InstrumentedEnqueueRequestForObject struct {
 
 // Create implements EventHandler, and creates the metrics.
 func (h InstrumentedEnqueueRequestForObject) Create(e event.CreateEvent, q workqueue.RateLimitingInterface) {
-	setResourceMetric(e.Meta, e.Object)
+	setResourceMetric(e.Object, e.Object)
```

Comment:
Change these function's signatures to only take one object if possible
```suggestion
	setResourceMetric(e.Object)
```